backend/src/main/java/pl/masi/service/UserService.java

```diff
@@ -86,8 +86,17 @@ public void addUser(RegistrationRequestDto userEntity) {
         user.setRole(User.Role.CANDIDATE);
         user.setLanguage(userEntity.getLanguage());
         this.userRepository.saveAndFlush(user);
+    }
 
-
+    @Transactional
+    public void addRedactor(RegistrationRequestDto userEntity) {
```

Comment:
Ta metoda może być wywoływana tylko przez moderatora, tak? Jeśli tak to powinna być oznaczona `    @PreAuthorize("hasRole('ROLE_MODERATOR')")`